package/src/components/Attachment/__tests__/Giphy.test.js

```diff
@@ -291,4 +267,39 @@ describe('Giphy', () => {
       expect(queryByTestId('giphy-attachment')).toBeTruthy();
     });
   });
+
+  it('should render a loading indicator in giphy image and when successful render the image', () => {
+    const { getByA11yLabel, getByAccessibilityHint } = render(
+      <OverlayProvider>
+        <Chat client={chatClient}>
+          <Channel channel={channel}>
+            <MessageList />
+          </Channel>
+        </Chat>
+      </OverlayProvider>,
+    );
+
```

Comment:
can you expect that the `getByAccessibilityHint('image-loading')` will throw because it's non existing in the start?

Reply:
But it exists at the start

Reply:
But it exists at the start. The hook that displays the loading indicator is default to true at the start